packages/manager/modules/pci/src/projects/project/storages/cloud-archives/cloud-archives.routing.js

```diff
@@ -9,6 +9,7 @@ export default /* @ngInject */ ($stateProvider) => {
         dynamic: true,
         type: 'string',
       },
+      trackingTag: null,
```

Comment:
**nitpick(issue):** Adding a state param without adding it in url is not a good solution as we cannot have consistent behavior on refresh

Reply:
I am not sure any issue will happened.
In this case we would not persist the tracking if you reload the page.

The tracking has to be triggered only if we coming from a success task.